config/samples/app_v1beta1_application.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     matchLabels:
       app.kubernetes.io/name: "wordpress-01"
   componentKinds:
-  - group: core
+  - group: v1
```

Comment:
I understand this change. Why change "" to v1 in the others ?

Reply:
per Janet's comment: https://github.com/kubernetes-sigs/application/issues/112#issuecomment-580008476.
It seems `v1` makes more sense than `""`.